go_/models/player.go

```diff
@@ -0,0 +1,48 @@
+package models
+
+import (
+    "bufio"
+    "fmt"
+    "strings"
+    "strconv"
+)
+
+type Player struct {
+    name string
+    token string
+}
+
+func (p *Player) Name() string {
+    return p.name
+}
+
+func (p *Player) Prompt() {
+    fmt.Printf("%s, place your \"%s\": ", p.name, p.token)
+}
+
+func (p *Player) PlayToken(r *bufio.Reader, board *Board) (row int, col int) {
+    var err error
+
+    for {
+        p.Prompt()
+        choice, _ := r.ReadString('\n')
+        col, err = strconv.Atoi(strings.TrimSpace(choice))
+
+        if err != nil { continue }
+
+        col--
+        row, err = board.Receive(col, p.token)
+
+        if err == nil { break }
+    }
+
+    return
+}
```

Comment:
These things probably shouldn't be implemented on `*Player`, it was just fun to play around with the module/subpackage system